### ISSUE TITLE
Предлагаю в скриптах переписать звук использования ЛК у Е-образных и возможно Дшки от 717 на свтй

### ISSUE THREAD

**Issue by Alexrig23** [metrostroi-repo/MetrostroiAddon#385]

**Comment by CrIcKeT98** [comment]
Откуда взяты эти звуки?

**Comment by Alexrig23** [comment]
> Откуда взяты эти звуки?

Мне знакомый дал.

**Comment by Alexrig23** [comment]
> Откуда взяты эти звуки?

И он разрешил их использовать для обновления.

**Comment by CrIcKeT98** [comment]
Ну а это его звуки? Просто похоже, что они из другой игры

**Comment by Alexrig23** [comment]
> Ну а это его звуки? Просто похоже, что они из другой игры

Да, его звуки, это не из игры, он сам записывал вблизи когда состав стоял на станции перед отправлением.

**Comment by Alexrig23** [comment]
Кстати вот ещё два варианта звуков ЛК для 81-717 всё от того же знакомого, он разрешает использовать для обновления. https://drive.google.com/file/d/1V5r2UhXqTrEqApZ7JpYvDJiLZ-DpIe5B/view?usp=sharing

**Comment by Alexrig23** [comment]
https://drive.google.com/file/d/1PYczuxh-zikT7K64pC7_JuGAX39uLr7f/view?usp=sharing + обновлённый звук ЛК (фикс) для Еж3 и остальных Е-образных ну и звук РК свой для них.

**Comment by Alexrig23** [comment]
https://drive.google.com/file/d/1HMfp4wyeN99qSqdRqtOKcGByyvspxg-T/view?usp=sharing вот ещё обновлённый звук ЛК (фикс) для Еж3 и остальных Е-образных ну и новый звук РК для них.

**Comment by Alexrig23** [comment]
ну что там по звукам, подумали?

**Comment by Alexrig23** [comment]
https://drive.google.com/file/d/1ERB4TSxX47g38sUGJD0W9KU1WGtLUmSC/view?usp=sharing второй вариант звучания ЛК для Еж3, было записано на Новогоднем Еж3.

**Comment by Alexrig23** [comment]
@HellReach мб вы возьмёте и перепишите пожалуйста? Просто уже пора закрывать эту тему и внести звуки ЛК и РК для Е-образных.

**Comment by Alexrig23** [comment]
@Hellss мб вы возьмёте и перепишите пожалуйста? Просто уже пора закрывать эту тему и внести новые звуки ЛК и РК для Е-образных.

**Comment by CrIcKeT98** [comment]
Ответ Глеба:
"
https://drive.google.com/file/d/1FTG3XSPTe1EVNV4WlLHsKmxvL4IJ3tMY/view?usp=sharing
Хорошо звучит только разбор, и то с кусочком другого аппарата

https://drive.google.com/file/d/1V5r2UhXqTrEqApZ7JpYvDJiLZ-DpIe5B/view?usp=sharing
Звучит неплохо, надо посмотреть насколько оно будет подходить к текущему сбору схемы

https://drive.google.com/file/d/1HMfp4wyeN99qSqdRqtOKcGByyvspxg-T/view?usp=sharing
Не понял отличия. РК похож на метростроевский по памяти

https://drive.google.com/file/d/1ERB4TSxX47g38sUGJD0W9KU1WGtLUmSC/view?usp=sharing 
Порезано заметно, какой то хвост/шум в конце. Не уверен что это будет вписываться в игру

Кроме того, всё же РУ1 и обычный Еж3 может отличаться по аппаратам
"

**Comment by Alexrig23** [comment]
> Ответ Глеба: " https://drive.google.com/file/d/1FTG3XSPTe1EVNV4WlLHsKmxvL4IJ3tMY/view?usp=sharing Хорошо звучит только разбор, и то с кусочком другого аппарата
> 
> https://drive.google.com/file/d/1V5r2UhXqTrEqApZ7JpYvDJiLZ-DpIe5B/view?usp=sharing Звучит неплохо, надо посмотреть насколько оно будет подходить к текущему сбору схемы
> 
> https://drive.google.com/file/d/1HMfp4wyeN99qSqdRqtOKcGByyvspxg-T/view?usp=sharing Не понял отличия. РК похож на метростроевский по памяти
> 
> https://drive.google.com/file/d/1ERB4TSxX47g38sUGJD0W9KU1WGtLUmSC/view?usp=sharing Порезано заметно, какой то хвост/шум в конце. Не уверен что это будет вписываться в игру
> 
> Кроме того, всё же РУ1 и обычный Еж3 может отличаться по аппаратам "


https://drive.google.com/file/d/1TEC7f6uXi0L2OAlv94o2rxEfXhvEjz3u/view?usp=sharing вот ещё 3 варианта звучания ЛК + звук тележек на разных скоростях для 81-717, также ещё токоразделы они универсальные, если можете то попробуйте реализовать рандомизацию звуков ЛК пожалуйста.

про РК, звук был совмещен с Номерным и Ежом т.к. он в реале у Еж3 и остальных Е-образных так и звучит.
